app/services/classification_service.py

```
@@ -0,0 +1,416 @@
+            except ValueError:
+                return None
+        return None
+
+    def _row_to_classification(self, row: Dict[str, Any]) -> Classification:
+        """Convert a database row to a Classification object"""
+        keywords = row.get("keywords_matched", [])
+        if isinstance(keywords, str):
+            import json
+            try:
+                keywords = json.loads(keywords)
+            except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)